src/Symfony/Component/Notifier/Bridge/Gitter/composer.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Can you please use the same order of the keys as in the other bridges?
```json
{
    "name": "symfony/sendinblue-notifier",
    "type": "symfony-bridge",
    "description": "Symfony Sendinblue Notifier Bridge",
    "keywords": ["sms", "sendinblue", "notifier"],
    "homepage": "https://symfony.com",
    "license": "MIT",
    "authors": [
        {
            "name": "Pierre Tondereau",
            "email": "pierre.tondereau@protonmail.com"
        },
        {
            "name": "Symfony Community",
            "homepage": "https://symfony.com/contributors"
        }
    ],
    "require": {
        "php": ">=7.2.5",
        "ext-json": "*",
        "symfony/http-client": "^4.3|^5.0",
        "symfony/notifier": "~5.2.2"
    },
    "autoload": {
        "psr-4": { "Symfony\\Component\\Notifier\\Bridge\\Sendinblue\\": "" },
        "exclude-from-classmap": [
            "/Tests/"
        ]
    },
    "minimum-stability": "dev"
}
```

also php requirement should be added: `"php": ">=7.2.5",`

Reply:
Done :+1: